lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -181,6 +182,48 @@ def test_duplicate(self, selinux_disabled, server_config, make_logger, tarball):
             tree.create("ABC", source_tarball)
         assert exc.value.tarball == Dataset.stem(source_tarball)
 
+    def test_unpack(
+        self, monkeypatch, selinux_enabled, server_config, make_logger, tarball
+    ):
+        """Test to check the unpacking functionality of the FileTree"""
+        source_tarball, source_md5, md5 = tarball
+
+        def mock_run(args, cwd, stderr):
+            return subprocess.CompletedProcess(
+                args, returncode=0, stdout=b"Some error running tar", stderr=None
```

Comment:
I think you should drop the `stdout` here, or at least change it to something other than "Some error", as this is supposed to be the success case!